BasicSample/Assets/Rendering/Scripts/StereoSeparationSlider.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.MixedReality.OpenXR.Samples;
+using Microsoft.MixedReality.Toolkit.UI;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    public class StereoSeparationSlider : MonoBehaviour
+    {
+        [SerializeField]
+        private GameObject m_pinchSlider = null;
```

Comment:
nit: not sure why but we don't usually name serialized fields using "m_" prefix.

Reply:
Done